api/controllers/console/datasets/hit_testing_base.py

```diff
@@ -42,6 +42,11 @@ def get_and_validate_dataset(dataset_id: str):
     def hit_testing_args_check(args):
         HitTestingService.hit_testing_args_check(args)
 
+    @staticmethod
+    def retrieval_args_check(args):
```

Comment:
`hit-testing` is the same with `retrieve`.

- `retrieve` is the new endpoint
- We keep the `hit-testing` mainly because we do not want to break existing applications.